### ISSUE TITLE
How to change a user's role in WorkSpace in MySQL?

### ISSUE THREAD

**Issue by dwpoint** [ONLYOFFICE/CommunityServer#468]
Hello! I have created administrator accounts. But I would like to make them regular users. Is it possible to change their role via MySQL? I just didn't find the right table in the database.

**Comment by dwpoint** [comment]
Simply put, I want to understand where the user's role is stored in the WorkSpace. The role of owner, administrator, standard user, and guest. MySQL does not have this.

**Comment by Carazyda** [comment]
Hello @dwpoint you can find this infromation in core_usergroup table.
Admins - `SELECT * FROM core_usergroup WHERE groupid='cd84e66b-b803-40fc-99f9-b2969a54a1de';`
Guests - `SELECT * FROM core_usergroup WHERE groupid='aced04fa-dd96-4b35-af3e-346bf1eb972d';`

For the user type - no group is allocated. Only created already on the portal.

Owner you can find in tenants_tenants table.

**Comment by dwpoint** [comment]
> 

@Carazyda With the administrator and owner it is clear. Thank you very much.

But I'm not quite sure about regular users. Are you saying that when creating a regular user, the portal will generate the group ID itself, or do I need to create it?

I just remember creating a regular user, but I did not create a group at the same time. And now I've assigned an administrator to this user.

As the owner, can I remove the administrator rights from the user and make them a regular group member?

**Comment by Carazyda** [comment]
No, portal not created a group by default for users. Only if you did it manually in the People module.

You can remove administrator rights in the portal settings Security->Access Rights(/Management.aspx?type=4) by uncheck checkbox on Full Access.

**Comment by dwpoint** [comment]
@Carazyda Good. But if I give a person a link to register. After registration, then he will be a guest?

**Comment by Carazyda** [comment]
If you have generated an invitation link for guests (check "Add users as guests") - yes, they will be guests. Default invite link for users.

**Comment by dwpoint** [comment]
@Carazyda Yes, I'm talking about the default link. If a user registers using it, what groupid will it be?

**Comment by Carazyda** [comment]
For regular users, there is no default group and no entry in the table core_usergroup.

**Comment by dwpoint** [comment]
@Carazyda Understood. Thank you very much! Do you have time to look at a couple of my other questions on ussues CommunityServer?
